### PR TITLE
Storing ingested data as HTML instead of Markdown

### PR DESCRIPTION
**Pull request by clintonb**
This removes the burden of conversion from our API clients.

ECOM-5623

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review

**Comment by rlucioni** [comment]
@clintonb do we already have test coverage for this? If not, could you please add a test for this?

File: course_discovery/apps/course_metadata/data_loaders/__init__.py [hunk 1]
```diff
@@ -82,12 +85,19 @@ def clean_strings(cls, data):
...
+        attributes), and converts the Markdown back to HTML.
+        """
+        cleaned = content.replace('&nbsp;', '')
         html_converter = html2text.HTML2Text()
         html_converter.wrap_links = False
         html_converter.body_width = None
-        return html_converter.handle(stripped).strip()
+        cleaned = html_converter.handle(cleaned).strip()
+        cleaned = markdown.markdown(cleaned)
+        cleaned = cls.MARKDOWN_CLEANUP_REGEX.sub(r'\1', cleaned)
```
Comment (rlucioni) [inline course_discovery/apps/course_metadata/data_loaders/__init__.py:99]: This works. However, in the interest of less surprising, more readable code, what about doing this without the regex? You can define a small helper method:

``` py
def strip_p_tags(html):
    return html.lstrip('<p>').rstrip('</p>')
```

Comment (clintonb) [reply course_discovery/apps/course_metadata/data_loaders/__init__.py:99]: The strip methods on strings will treats the arguments as a list of characters to remove, not a word. In some cases, like `<em>TOEFL</em>`, the opening angle bracket is removed.

**Comment by schenedx** [comment]
👍

**Comment by mikedikan** [comment]
👍

**Pull request merged**